webapp/src/config/general.js

```diff
@@ -5,3 +5,5 @@ export const eosRateLink = process.env.REACT_APP_EOS_RATE_LINK
 export const defaultProducerLogo =
   process.env.REACT_APP_DEFAULT_PRODUCER_LOGO ||
   'https://bloks.io/img/eosio.png'
+export const appVersion = process.env.REACT_APP_VERSION || 'v1.0'
+export const appName = process.env.REACT_APP_NAME || 'eosiodasboard'
```

Comment:
typo : `eosiodasboard` => `eosiodashboard `

Reply:
nice catch, fixed in my last commit